src/components/HeroSection/HeroSection.tsx

```diff
@@ -11,20 +11,41 @@ interface HeroSectionProps {
   image: string
   altText: string
   isFlipped: boolean
-  buttonText: string
-  redirectTo: string
+  buttonText?: string
+  redirectTo?: string
 }
 
-export const HeroSection = ({ headerText, bodyText, image, altText, isFlipped, buttonText, redirectTo }: HeroSectionProps): JSX.Element => {
+export const HeroSection = ({
+  headerText,
+  bodyText,
+  image,
+  altText,
+  isFlipped,
+  buttonText,
+  redirectTo
+}: HeroSectionProps): JSX.Element => {
+  const redirectObj = {
```

Comment:
had to make this an object because next's `Link` component requires a URL object instead of a string